Makefile

```diff
@@ -181,10 +181,6 @@ errcheck:
 	fi
 	errcheck $(PACKAGES)
 
-.PHONY: lint
-lint:
```

Comment:
I'd prefer to make `lint` an alias of `golangci-lint` which I find too long to type.. :smirk

Reply:
Yeah, I'm thinking about re-adding `lint` but it would run the JS linter too 😉